parent/pom.xml

```diff
@@ -41,8 +41,8 @@
 
         <ant.version>1.9.5</ant.version>
         <assertj.core.version>3.1.0</assertj.core.version>
-        <assertj.generator.version>1.6.0</assertj.generator.version>
-        <avro.version>1.7.7</avro.version>
+        <assertj.generator.version>2.0.0</assertj.generator.version>
+        <proto.version>3.0.0-beta-2</proto.version>
```

Comment:
Please update this to 3.0.0-beta-3 ASAP - it's the only thing that's holding up my +1